charms/kfp-profile-controller/tox.ini

```diff
@@ -33,7 +33,9 @@ allowlist_externals =
     xargs
 commands = 
 ; uses 'bash -c' because piping didn't work in regular tox commands
-  bash -c 'find . -type f -name "requirements*.in" | xargs --replace=\{\} pip-compile -U --resolver=backtracking \{\}'
+  pip-compile requirements.in
+  pip-compile requirements-fmt.in
+  bash -c 'find . -type f -name "requirements*.in" | xargs --replace=\{\} pip-compile --resolver=backtracking \{\}'
```

Comment:
If you're editing this, can you use [this value](https://github.com/canonical/kfp-operators/pull/284/files#diff-402bba52aa5970db56379017309e9943cb8c9bb3e40f6a98843c55745a5049fdR34) so we don't have a conflict?